api/src/pcapi/core/users/external/batch.py

```diff
@@ -31,6 +31,7 @@ def format_user_attributes(user_attributes: UserAttributes) -> dict:
         "u.is_beneficiary": user_attributes.is_beneficiary,
         "date(u.deposit_expiration_date)": _format_date(user_attributes.deposit_expiration_date),
         "date(u.last_booking_date)": _format_date(user_attributes.last_booking_date),
+        "u.roles": [role.value for role in user_attributes.roles],
```

Comment:
il faut utiliser la notion de `tag` : `https://doc.batch.com/api/custom-data-api/set-update`, (cf ut.booking_categories)

Est ce que tu peux tester que le commentaire "A Batch tag can't be an empty list, otherwise the API returns an error" est toujours vrai ?

Si oui, lorsqu'on a une liste vide, est-ce qu'il faut retourner None alors ? au cas où on passe de ['BENEFICIARY'] à [] ?

(ce qui n'est pas fait pour les `ut.booking_categories` au passage...)